PeerioTechnologies/peerio-desktop
# Misc refactors and cleanup

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/4282/desktop-inlinefiles-jsx-nesting-level-is-too-much-refactor-into-smaller-components
https://app.clubhouse.io/peerio/story/17551/desktop-refactor-url-preview-into-two-separate-components-image-website
https://app.clubhouse.io/peerio/story/17977/desktop-no-more-need-for-long-emojiimage-css-comment
https://app.clubhouse.io/peerio/story/11579/make-contactprofile-api-similar-to-dialog

#### Testing instructions
This branch has a bunch of refactors and cleanup that shouldn't functionally be any different from the `dev` branch. Here are the components that are affected, just need to double check that they aren't behaving any different from before:
* Inline files
* URL previews
* ContactProfile -- the dialog with more info about a contact that pops up when you click on these locations:
  * username when a user gets `@` mentioned in a message
  * username of invited user in the "Members" section in chat sidebar
  * name of the owner in Files view
  * most avatars (don't need to check them all; if it's working then it's working, it's the same component everywhere)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
